mass_mailing_autoresponder_crm_import/wizard/import_knime_export_wizard.py

```diff
@@ -0,0 +1,191 @@
+import base64
+from datetime import datetime, timedelta
+import csv
+import io
+import logging
+
+from odoo import models, fields, api, _
+from odoo.exceptions import UserError
+from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT
+from odoo.modules.module import get_resource_path
+import pytz
+from xlrd import open_workbook
+
+_logger = logging.getLogger(__name__)
+
+
+class ImportKnimeExportWizard(models.TransientModel):
+    _name = 'import.knime.export.wizard'
+    _description = 'Import Knime Export'
+
+    file = fields.Binary(string='File')
+    filename = fields.Char(string='File name')
+    segment = fields.Char()
+    email = fields.Char()
+    active_mail = fields.Char()
+    send_today = fields.Boolean()
+
+    file = fields.Binary(string='File')
+    filename = fields.Char(string='File name')
+    dwnld_csv_filename = fields.Char("CSV File name")
+    dwnld_xls_filename = fields.Char("XLS File name")
+    dwnld_txt_filename = fields.Char("TXT File name")
+    dwnld_csv_file = fields.Binary("CSV File")
+    dwnld_xls_file = fields.Binary("XLS File")
+    dwnld_txt_file = fields.Binary("TXT File")
+
+
+    @staticmethod
+    def check_header(header):
+        """
+        Verify that header has the required columns as well as create
+         a dict with indexes.
+         """
+        # Verify Header, Force it lowercase
+        header = {x.lower(): index for index, x in enumerate(header)}
+        correct_header = ['e-postadress', 'segment', 'sendtoday', 'activemail']
+        if not all(item in header for item in correct_header):
+            raise UserError(_('Please correct Header in file!\n'
+                              'Header has to contain:\n') +
+                            '\n'.join(correct_header))
+        return header
+
+
+    def import_row(self, email, segment, send_today,
+                   active_email, partner_obj, campaign_obj):
+        """Handle an individual row."""
+        partner = partner_obj.search([('email', '=', email)], limit=1)
+        if not partner:
+            raise UserError(
+                _('Can not find jobseeker with email: {}').format(email))
+        if segment == 'a' or segment == 'b' or segment == 'c':
```

Comment:
Error message refers to A,B,C and not the lowercase a,b,c is this correct?
An alternative:
if segment in ('a', 'b', 'c'):

Reply:
Changed error message to lower case and used your alternative